test/modules/common/admin-container-test.js

```diff
@@ -7,13 +7,16 @@ import React from 'react';
 import sinon from 'sinon';
 import { expect } from 'chai';
 import { shallow, mount } from 'enzyme';
+import FakeLocalStorage from '../../utils/fake-local-storage';
 import { AdminContainer } from '../../../src/modules/common/containers/admin-container';
 
 const user = {
   id: '34',
   admin: true
 };
 
+const storage = window.localStorage || new FakeLocalStorage();
```

Comment:
I think the latest version of JSDOM has a mock for local storage built in now, so this fallback shouldn't be necessary.